app/src/js/components/Pdr/pdr.js

```diff
@@ -168,11 +168,12 @@ class PDR extends React.Component {
     const deleteStatus = get(pdrs.deleted, [pdrName, 'status']);
     const { error } = record;
 
-    const granulesCount = get(record, 'data.granulesStatus', []);
+    const granulesCount = get(record, 'data.stats', []);
```

Comment:
Do we know that the .stats response is accurate for these numbers? I know there had been some question about that a while ago

Reply:
I don't know if the numbers are accurate in all cases.  But I got a couple of completed PDRs, the numbers (`total` and `completed`) are there.